src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -14,13 +14,14 @@ The .NET Foundation licenses this file to you under the MIT license.
 -->
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
+    <_IsApplePlatform Condition="'$(TargetOS)' == 'osx' or $(TargetOS.StartsWith('ios'))">true</_IsApplePlatform>
```

Comment:
Should this also handle `tvos`, `tvossimulator`, and `maccatalyst`? PR #81780 enables the build for them.

Reply:
We could, but we have to think at which point we would set up the limitation of which platforms are supported and available to the end user through: the build system, available runtime packages, SDK, etc, before committing on supporting all of them. 
@marek-safar what do you think?

Reply:
Right, it make sense to add them as well. On which level there should be a check for unsupported runtime identifiers?

Reply:
I'm personally interested in `maccatalyst` because it makes it much easier to test the runtime itself (no simulators or devices necessary).

Reply:
We are going to mark them all experimental for now so why not have them all?

Reply:
Alright, should we then update the parent tracking issue (and relevant issues) title/description about this?

Reply:
I'd just list these platforms somewhere in P1/P2 tasks for verification.